src/status_im/subs/wallet/wallet.cljs

```diff
@@ -286,6 +286,13 @@
  :<- [:wallet/create-account]
  :-> :selected-keypair-uid)
 
+(rf/reg-sub
+ :wallet/selected-keypair-keycard?
+ :<- [:wallet/selected-keypair]
+ (fn [selected-keypair]
+   (let [{:keys [keycards]} selected-keypair]
```

Comment:
Here the destructuring can happen on the function binding instead of the let binding.

It's slightly better for performance to use as input a subscription that only returns the `:keycards` value from `:wallet/selected-keypair`, not the whole `:wallet/selected-keypair`.